packages/core/src/Core.ts

```diff
@@ -78,19 +78,25 @@ export class Core<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
             const previousSibling = range.start.previousSibling();
             if (previousSibling) {
                 previousSibling.removeBackward();
-            } else if (range.startContainer.breakable) {
+            } else if (range.startContainer.breakable && !range.startContainer.independent) {
```

Comment:
I feel this is outside of the scope of this PR so I'm not reviewing this commit, especially considering it uses the `independent` concept that we are still unsure of. (Yesterday we talked about `editable`.)